src/Entity/User.php

```diff
@@ -183,7 +183,7 @@ public function getAvailableAssignmentById(int $assignmentId): Assignment
     {
         $assignment = $this->getAssignmentById($assignmentId);
 
-        if ($assignment->isAvailable()) {
+        if ($assignment->getLineItem()->isActive() && $assignment->isAvailable()) {
```

Comment:
This solves when we call `/api/v1/assignments/1/lti-link` (by ID) but we also need to change when we list all assignments for the `/api/v1/assignments` endpoint.

Right now it still brings it as `ready`:

```
{
    "assignments": [
        {
            "id": 1,
            "username": "01X9830002K0001",
            "state": "ready",
            "attemptsCount": 9,
            "lineItem": {
                "uri": "https://depp.fr#February20-Delivery_20auseva2_fr",
                "label": "20auseva2_fr",
                "startDateTime": 1606780800,
                "endDateTime": 1607126420,
                "maxAttempts": 0
            }
        }
    ]
}
```